examples/text_summarization/extractive_summarization_cnndm_transformer.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>Set USE_PREPROCSSED_DATA to True by default?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/nlp-recipes/pull/515/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Prefer to use False since it shows the complete flow.

Reply:
Yeah. That's fine. Just realized that QUICK_RUN also helps preventing the notebook running for 28 hours.